plugins/api-docs/src/components/ApiExplorerTable/ApiExplorerTable.tsx

```diff
@@ -151,11 +151,9 @@ export const ApiExplorerTable = ({
 
   if (error) {
     return (
-      <div>
-        <Alert severity="error">
-          Error encountered while fetching catalog entities. {error.toString()}
-        </Alert>
-      </div>
+      <WarningPanel severity="error" title="Could not fetch catalog entities.">
+        <code>{error.toString()}</code>
```

Comment:
Or `<CodeSnipped>` from `@backstage/core`?

Reply:
Wasn't aware of this!

Reply:
Looking at that component, it's primarily for wrapping code snippets based on language formatting?  In this example, it'll always just be raw error codes and HTTP, never a language. Is `<CodeSnippet>` still appropriate?

The project as a whole has many other references to the raw JSX HTML element `<code>`, so thinking in this instance it makes sense to use that? 

![image](https://user-images.githubusercontent.com/33203301/107085209-1ea13900-67c6-11eb-8bbb-e74e796256ec.png)

Reply:
You could use the code snipped with `language="text"`

Reply:
Note, CodeSnippet works, but it adds a lot of extra spacing around the attribute. (though it does support line breaks in the error msg, such as the wraps in a stack trace, so that's nice)

Anyway, happy to go with it for now based on your feedback, will push commit shortly.

![image](https://user-images.githubusercontent.com/33203301/107243970-87252b80-69fb-11eb-8ded-81e3afde0931.png)